openff/benchmark/analysis/new_draw.py

```diff
@@ -0,0 +1,973 @@
+    results = {}
+    for path in results_dir:
+        if (os.path.isfile(path) and path.split('.')[-1].lower() == 'csv'):
+            method = '.'.join(os.path.split(path)[1].split('.')[:-1])
+            results[method] = pd.read_csv(path)
+        elif os.path.isdir(path):
+            for root, dirs, files in os.walk(path):
+                for file in files:
+                    path = os.path.join(root, file)
+                    if (os.path.isfile(path) and path.split('.')[-1].lower() == 'csv'):
+                        method = '.'.join(file.split('.')[:-1])
+                        results[method] = pd.read_csv(path)
+
+    # apply the intersection method
+    for m, df in results.items():
+        results[m].set_index('mm_conf', inplace=True)
+
+    index_intersect = results[method].index
+    for m in results:
+        index_intersect = index_intersect.intersection(results[m].index)
+    for m, df in results.items():
+        results[m] = df.loc[index_intersect]
+        if results[m].shape != df.shape:
+            warnings.warn(f"Not all conformers of method {m} considered, because these are not available in other methods.")
+
+    combined_df = pd.concat(results, names=['method']).reset_index(0)
+
+    new = combined_df[(combined_df['rmsd (qm_min / mm_conf)']<float(rmsd_cutoff))]
+    new = new.reset_index()
+    new.drop('mm_conf', axis='columns', inplace=True)
+    new = new[['qm_min', 'mm_min', 'rmsd (qm_min / mm_conf)', 'dE (mm_conf - mm_min)', 'method']]
+    new = new.rename(columns={'rmsd (qm_min / mm_conf)': 'rmsd ($\AA$)', 'dE (mm_conf - mm_min)': 'dE (kcal/mol)'})
+
+    new2 = combined_df[(combined_df['dE (mm_conf - mm_min)']<float(de_cutoff))]
+    new2 = new2.reset_index()
+    new2.drop('mm_conf', axis='columns', inplace=True)
+    new2 = new2[['qm_min', 'mm_min', 'rmsd (qm_min / mm_conf)', 'dE (mm_conf - mm_min)', 'method']]
+    new2 = new2.rename(columns={'rmsd (qm_min / mm_conf)': 'rmsd ($\AA$)', 'dE (mm_conf - mm_min)': 'dE (kcal/mol)'})
+
+    draw_ridge_swope(
+        results,
+        new,
+        'dE (mm_conf - mm_min)',
+        'dE (kcal/mol)',
+        out_file=os.path.join(output_directory, f'fig_ridge_de_rmsd_cutoff_{rmsd_cutoff}.png'),
+        what_for="talk",
+        bw="hist",
+        same_subplot=True,
+        sym_log=False,
+        hist_range=(-1.67,15)
+    )
+
+    draw_ridge_swope(
+        results,
+        new2,
+        'rmsd (qm_min / mm_conf)',
+        'rmsd ($\AA$)',
+        out_file=os.path.join(output_directory, f'fig_ridge_rmsd_de_cutoff_{de_cutoff}.png'),
+        what_for="talk",
+        bw="hist",
+        same_subplot=True,
+        sym_log=False,
+        hist_range=(0, 3)
+    )
+
+
+def draw_ridge_swope(
+    results,
+    dataframes,
+    key,
+    x_label,
+    out_file,
+    what_for="paper",
+    bw="hist",
+    same_subplot=False,
+    sym_log=False,
+    hist_range=(-15, 15)
+):
+    # Define and use a simple function to label the plot in axes coordinates
```

Comment:
Can you add a docstring indicating how this function differs from `draw.draw_ridge`?